### DISCUSSION TITLE
Diagnostic screen blank

### DISCUSSION THREAD

**Discussion by cykeler** [netzero-labs/netzero#30] [Q&A]
Hello , wondering if anyone has come accross a blank diagnositic screen in the app. I’m thinking there was an issue when first trying to connect to my pw3 , but the diagnostics screen has always been blank for me. I tried quitting the app, signing out and back in, but no luck. 
thanks !

**Comment by zigam** [comment]
There's a known issue with PW3 after scanning the QR code. If you sign out of the app and back in, that should restore the estimated capacity at least, does it not?

**Comment by cykeler** [comment]
Yes, I do see that, I was thinking it also displayed MPPT information?

Thanks so much for your help.
-Russell

**Reply by zigam** [reply]
Tesla made some changes to the underlying APIs, so I don't think PW3 wifi connection works currently (and I don't have access to a PW3 to test). I would suggest the Tesla One app in the meantime.

**Comment by cykeler** [comment]
Thanks for the tip ZIga! that's been my backup.

-Russell
